contracts/lease/src/api/query.rs

```diff
@@ -10,7 +10,10 @@ use super::{DownpaymentCoin, LeaseCoin, LpnCoin};
 
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
 #[serde(rename_all = "snake_case")]
-pub struct StateQuery {}
+pub enum QueryMsg {
+    State {},
```

Comment:
I can always rename the field through `serde`. I think that might be even more elegant solution, if we don't want breaking changes.

Reply:
I was also about to recommend this ... however, it seems reasonable to migrate it to `State`

Reply:
I have bad news though. The previous query's definition was actually just "object" without specifying any fields, thus there is no way to avoid the breaking change.

Reply:
Update: Left is as state.